sdk/communication/azure-communication-rooms/swagger/README.md

```diff
@@ -38,8 +37,28 @@ service-interface-as-public: true
 generate-sync-async-clients: false
 sync-methods: all
 add-context-parameter: true
+url-as-string: true
 context-client-method-parameter: true
 required-parameter-client-methods: true
 custom-strongly-typed-header-deserialization: true
 generic-response-type: true
 ```
+ 
+### Rename RoomModal to CommunicationRoom
+```yaml
+directive:
+  - from: swagger-document
+    where: $.definitions.Role["x-ms-enum"]
+    transform: >
+      $.name = "ParticipantRole";
+```
+
+
+### Rename RoomModal to CommunicationRoom
```

Comment:
Do you want to rename roomModel to communication room? that will prevent you from having to do the conversion in your clients. That will make your code cleaner

If so, please rename to `RoomModel` and rerun autorest.

Reply:
No, I was experimenting with renaming and left this here. This didn't have any effect. Removed now in latest commit.